src/probnum/_config.py

```diff
@@ -4,24 +4,24 @@
 
 class Configuration:
     """
-    Configuration over which some mechanics of probnum can be controlled dynamically.
+    Configuration over which some mechanics of ProbNum can be controlled dynamically.
 
-    ``ProbNum`` provides some configurations together with default values. These
+    ProbNum provides some configurations together with default values. These
     are listed in the tables below.
-    Additionally, the user can register own configuration entries via the method
+    Additionally, users can register their own configuration entries via
     :meth:`register`. Configuration entries can only be registered once and can only
     be used (accessed or overwritten) once they have been registered.
 
-    +----------------------------------+-----------------------------------------------+
-    | Config entry                     | Description                                   |
-    +==================================+===============================================+
-    | ``covariance_inversion_damping`` | A (typically small) value that is per         |
-    |                                  | default added to the diagonal of covariance   |
-    |                                  | matrices in order to make inversion           |
-    |                                  | numerically stable.                           |
-    +----------------------------------+-----------------------------------------------+
-    | ``...``                          | ...                                           |
-    +----------------------------------+-----------------------------------------------+
+    +----------------------------------+----------------------------------------------+
+    | Config entry                     | Description                                  |
```

Comment:
I sugges

```python
"""
+----------------------------------+---------------+-------------------------------------------------------------------------------------------------------------------------------------------+
| Config entry                     | Default value | Description                                                                                                                               |
+==================================+===============+===========================================================================================================================================+
| ``covariance_inversion_damping`` | 1e-12         | A (typically small) value that is per default added to the diagonal of covariance matrices in order to make inversion numerically stable. |
+----------------------------------+---------------+-------------------------------------------------------------------------------------------------------------------------------------------+
| ``...``                          | ``...``       | ...                                                                                                                                       |
+----------------------------------+---------------+-------------------------------------------------------------------------------------------------------------------------------------------+
"""
```

Reply:
I'm hacking together a quick sphinx extension that generates the documentation automatically atm.
I will include the default value column there.